src/SFA.DAS.LearnerDataMismatches.Web/Pages/Learner.cshtml.cs

```diff
@@ -25,10 +25,16 @@ public class LearnerModel : PageModel
         public string LearnerName { get; set; }
 
         public IEnumerable<string> DataLockNames =>
-            CollectionPeriods
-                .SelectMany(c => c.UniqueDataLockNames)
-                .Distinct()
-                .OrderBy(x => x);
+            NewCollectionPeriods
+            .SelectMany(x => x.DataLocks)
+            .Select(x => x.ToString())
+            .OrderBy(x => x);
+
+        public IEnumerable<DataLockHelpCentreLink> DataLockLinks =>
```

Comment:
Same here, the page is trying to eagerly load this and throws a fit if the collection is not populated.

Reply:
I have made sure the collections are never `null` and so don't cause `NullReferenceException`